package.json

```diff
@@ -1,6 +1,5 @@
 {
   "dependencies": {
-    "@prisma/client": "2.24.0-dev.27",
```

Comment:
This is a change I made to get rid of unexpected versions and Prisma installations being there. 
No idea what the side effects could be or where this might be used.